core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -565,6 +561,12 @@ public SystemSessionProperties(
                         "Use partial Distinct Limit",
                         true,
                         true),
+                enumProperty(
+                        MARK_DISTINCT_STRATEGY,
+                        "",
```

Comment:
Please add DISTINCT_AGGREGATIONS_STRATEGY below the existing entry for MARK_DISTINCT_STRATEGY instead of moving MARK_DISTINCT_STRATEGY. The diff looks confusing right now.